compliance/azure/ahub_manual/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.2
+
+- Bug fix
```

Comment:
Can you change to something that explains the fix?  consider "Fix formatting of email incident"  or something like this.

Reply:
Done!